Find a functionally equivalent code:android.content.res.TypedArray.length:COMMENT
Method Modifier: public      
Comment:/**
 * Return the number of values in this array.
 */

@@ -1,3 +1,6 @@
 {
+    if (mRecycled) {
+        throw new RuntimeException("Cannot make calls to a recycled instance!");
+    }
     return mLength;
 }

